content/_releases/22/36.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Product Update - v22.36
+description: Details of the product update v22.36 from 8th of September 2022.
+layout: rel
+version: "22.36"
+section: Product Updates
+category: product-updates
+releaseDate: 2022-09-08
+redirect_from:
+  - /releases/22.36.html
+---
+
+## Improvements and Updates
+
+### Credentials page: new design
+
+This is continuation of our effort to modernise and harmonise look and feel of the
+platform user interface. In this release we improved the Credentials page. When
+you navigate to Integrate > Credentials page you will see a similar view:
+
+{% include img.html max-width="80%" url="/assets/img/RN/2236/RN2236-credentials-page.png" title="Credentials Page" %}
+
+When you select a component your screen would look like the one below for Salesforce:
+
+{% include img.html max-width="80%" url="/assets/img/RN/2236/RN2236-credential.png" title="Component Credential" %}
+
+You can select 3 dots to open the menu to **Verify**, **Edit** or **Delete** the credentials
+or use **+** (plus) in the right above to create a new one.
+
+{% include img.html max-width="80%" url="/assets/img/RN/2236/RN2236-credential-mod.png" title="Credential Modification" %}
+
+Here is how each credential page looks when you Edit it:
+
+{% include img.html max-width="80%" url="/assets/img/RN/2236/EN2236-credential-edit.png" title="Edit the Credential" %}
+
+### Retrieve new sample button
+
+While designing the flow using the flow designer often we need to retrieve new
```

Comment:
While developing new integration flows it is often necessary to retrieve new sample data after each flow step is created. This was previously possible by navigating **Back**. In this release we have added a new button **Retrieve new sample from component** that allows you to create new sample data without navigating away from your work.